airbyte-integrations/connectors/source-zendesk-talk/unit_tests/test_streams.py

```diff
@@ -1,13 +1,14 @@
 #
 # Copyright (c) 2022 Airbyte, Inc., all rights reserved.
 #
-
+import random
+from unittest.mock import MagicMock
```

Comment:
This import is not required as mocker is a wrapper around unittes.mock (check my suggestion below)

Reply:
Done